src/home/homed-manager.c

```diff
@@ -1191,6 +1191,12 @@ static int manager_listen_notify(Manager *m) {
         if (r < 0)
                 return r;
 
+        /* Set SO_PASSCRED on the listening socket as well. It only needs to be set after accept(), but setting
+         * it early here can prevent race conditions in some cases */
+        int oc = 1;
+        if (setsockopt(fd, SOL_SOCKET, SO_PASSCRED, &oc, sizeof(oc)) < 0)
```

Comment:
already set literally in the previous line... ;-)

Reply:
Oh, oops.. My brain read `SO_PEERCRED` above.